docs/src/main/sphinx/release/release-435.md

```diff
@@ -0,0 +1,73 @@
+# Release 435 (13 Dec 2023)
+
+## General
+
+* Reduce coordinator memory usage. ({issue}`20018`, {issue}`20022`)
+* Improve insert reliability regarding memory consumption. ({issue}`20040`)
+* Fix incorrect results for `LIKE` with some strings containing repeated
+  substrings. ({issue}`20089`)
+* Fix coordinator memory leak. ({issue}`20023`)
+* Fix possible query failure for `MERGE` queries when `retry-policy` set to
+  `TASK` and `query.determine-partition-count-for-write-enabled` set to `true`.
+  ({issue}`19979`)
+* Prevent hanging query processing with `retry.policy` set to `TASK` when a
+  worker node died. ({issue}`18603 `)
+* Fix invalid position error when reading `ARRAY` columns. ({issue}`20065`)
+
+## Delta Lake connector
+
+* {{breaking}} Remove support for registering external tables with
+  `CREATE TABLE` and the `location` table property. Use the
+  `register_table` procedure as replacement. The property
+  `delta.legacy-create-table-with-existing-location.enabled` is
+  also removed. ({issue}`17016`)
+* Improve query planning performance on Delta Lake tables. ({issue}`19795`)
+* Ensure AWS access keys are used for connections to the AWS Security Token
+  Service. ({issue}`19982`)
+* Reduce memory usage for inserts into partitioned tables. ({issue}`19649`)
+
+## Elasticsearch connector
+
+* Ensure certificate validation is skipped when
+  `elasticsearch.tls.verify-hostnames` is `false`. ({issue}`20076`)
+
+## Hive connector
+
+* Add numeric integer to decimal coercion for Hive tables. ({issue}`19931`)
+* Add date to varchar coercion for Hive tables. ({issue}`19500`)
```

Comment:
These need to be rephrased in terms of schema evolution, as we've been doing for similar changes in older releases.

Reply:
This is how we used to write it in others .. but I updated to something better .. I think